index.bs

```diff
@@ -1462,3 +1484,231 @@ The [=remote end steps=] with |command parameters| are:
+
+ 1. If |context id| is null, return [=success=] with data null.
+
+ 1. If there is no browsing context with [=browsing context id=] |context id| return
+    [=error=] with [=error code=] [=no such frame=]
+
+ 1. Let |context| be the browsing context with id |context id|.
+
+ 1. Return [=success=] with data |context|
+
+</div>
+
+<div algorithm>
+To <dfn>get the browsing context info</dfn> given |context|:
+ 1. Let |context id| be the [=browsing context id=] for |context|.
+
+ 1. If |context| has a [=parent browsing context=] let |parent id| be the
+    [=browsing context id=] of that parent. Otherwise let |parent id| be null.
+
+ 1. Let |document| be |context|'s [=active document=].
+
+ 1. Let |url| be the result of running the [=URL serializer=], given
+    |document|'s [=spec=html/URL=].
+
+ 1. Let |context info| be a map matching the <code>BrowsingContextInfo</code>
+    production with the <code>context</code> field set to |context id|, the
+    <code>parent</code> field set to |parent id|, and the <code>url</code> field
+    set to |url|.
+
+ 1. Return |context info|.
+
+</div>
+
+### Commands ### {#module-browsingContext-commands}
+
+#### list ####  {#command-browsingContext-list}
+
+The <dfn export for=commands>browsingContext.list</dfn> command returns a list
+of all browsing contexts that are children of the given context, or all
+top-level contexts when no parent is provided.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      BrowsingContextListCommand = {
+        method: "browsingContext.list",
+        params: BrowsingContextListParameters
+      }
+
+      BrowsingContextListParameters = {
+        ?parent: BrowsingContext,
+      }
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl local-cddl">
+        BrowsingContextListResult = {
+          contexts: [*BrowsingContextInfo]
+        }
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for browsingContext.list">
+The [=remote end steps=] with |command parameters| are:
+
+  1. Let the |parent id| be the value of the <code>parent</code> field of
+       |command parameters| if present, or null otherwise.
+
+  1. Let |parent| be the result of [=trying=] to [=get a browsing context=]
+     given |parent id|.
+
+  1. If |parent| is null, let |contexts| be a list containing all [=top-level
+     browsing contexts=]. Otherwise let |contexts| be a list containing all
+     [=/browsing contexts=] which are [=child browsing contexts=] of |parent|.
+
+  1. Let |context info| be a list.
+
+  1. For each |context| of |context|, append the result of [=get the browsing
+     context info=] given |context| to |context info|.
+
+  1. Let |body| be a map matching the <code>BrowsingContextListResult</code>
+     production, with the <code>contexts</code> field set to |context info|.
+
+  1. Return [=success=] with data |body|.
+
+Issue: Should this contain other data like the current URL for the context.
```

Comment:
It does contain the current URL, doesn't it?

Reply:
Apparently I added that and forgot to remove this ;)